docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -2,67 +2,123 @@
 User mapping
 ============
 
-User mapping defines rules for mapping from users in the authentication system to Trino users.  This
-mapping is particularly important for Kerberos or certificate authentication where the user names
-are complex like ``alice@example`` or ``CN=Alice Smith, OU=Finance, O=Acme, C=US``.
+User mapping defines rules for mapping from users in the authentication method to Trino users. This
+mapping is particularly important for :doc:`Kerberos </security/kerberos>` or
```

Comment:
eros>`, or

Reply:
Disagree. There are only two items in the list, Kerberos auth or Certificate auth. A or B, not "A, or B" unless we mean to insert a pregnant pause in the reading after A.